Source/ACE.Entity/ObjectGuid.cs

```diff
@@ -5,14 +5,71 @@ namespace ACE.Entity
     public enum GuidType
     {
         Undef,
+        Weenie,
+        Static,
+        Generator,
         Player,
-        Creature
+        Creature,
+        Item
     }
 
     public struct ObjectGuid
     {
         public static readonly ObjectGuid Invalid = new ObjectGuid(0);
 
+        public static uint WeenieMin { get; } = 0x00000001;
+        public static uint WeenieMax { get; } = 0x000F423F;
+
```

Comment:
I'm not a huge fan of all of these ranges being tight.  The point of reading out of the DB is that we define a set of "acceptable" ranges, and then figure out what exists in the DB.  Its okay if the DB changes in the future, as long as everything is within acceptable ranges.   Tight ranges mean any change to the DB require changing this code.  I'm not super opposed to it, but I dislike having to change this every time we change the weenies in the DB.